client/src/components/Header/Header.jsx

```diff
@@ -1,40 +1,152 @@
-					</li>
-					<li>
-						<NavLink to="/about">About</NavLink>
-					</li>
-					<li>
-						<NavLink to="/services">Services</NavLink>
-					</li>
-					<li>
-						<NavLink to="/news">News</NavLink>
-					</li>
-					<li>
-						<NavLink to="/events">Events</NavLink>
-					</li>
-					<li>
-						<NavLink to="/gallery">Gallery</NavLink>
-					</li>
-					<li>
-						<NavLink to="/get-involved">Get Involved</NavLink>
-					</li>
-					<li>
-						<NavLink to="/contact-us">Contact Us</NavLink>
-					</li>
-				</ul>
-			</nav>
+			<div className="header-subcontainer">
+				<div className="header-logo-container">
+					<div className="header-logo-image-container">
+						<img
+							src="https://upload.wikimedia.org/wikipedia/commons/a/a8/Circle_Davys-Grey_Solid.svg"
+							alt="logo"
+						/>
+					</div>
+					<h2 className="header-logo-title">
+						Lentegeur Hospital
+						<br />
+						Facility Board
+					</h2>
+				</div>
+				<nav
+					className={`${isNavOpen ? "responsive_nav" : ""} ${
+						isClosingAdded ? "closing" : ""
+					}`}
+				>
+					<ul>
+						<li>
+							<NavLink to="/" className="navlink" onClick={closeNavbar}>
+								Home
+							</NavLink>
+							<div className="navlink-image-container">
+								<AiFillCaretUp />
+							</div>
+						</li>
+						<li>
+							<NavLink to="/about" className="navlink" onClick={closeNavbar}>
+								About LSH
+							</NavLink>
+							<div className="navlink-image-container">
+								<AiFillCaretUp />
+							</div>
+						</li>
+						<li>
+							<NavLink to="/services" className="navlink" onClick={closeNavbar}>
+								Services
+							</NavLink>
+							<div className="navlink-image-container">
+								<AiFillCaretUp />
+							</div>
+						</li>
+						<li>
+							<NavLink to="/news" className="navlink" onClick={closeNavbar}>
+								News
+							</NavLink>
+							<div className="navlink-image-container">
+								<AiFillCaretUp />
+							</div>
+						</li>
+						<li>
+							<NavLink to="/events" className="navlink" onClick={closeNavbar}>
+								Events
+							</NavLink>
+							<div className="navlink-image-container">
+								<AiFillCaretUp />
+							</div>
+						</li>
+						<li>
+							<NavLink to="/gallery" className="navlink" onClick={closeNavbar}>
+								Gallery
+							</NavLink>
+							<div className="navlink-image-container">
+								<AiFillCaretUp />
+							</div>
+						</li>
+						<li>
+							<NavLink
+								to="/get-involved"
+								className="navlink"
+								onClick={closeNavbar}
+							>
+								Get Involved
+							</NavLink>
+							<div className="navlink-image-container">
+								<AiFillCaretUp />
+							</div>
+						</li>
+						<li>
+							<NavLink
+								to="/contact-us"
+								className="navlink"
+								onClick={closeNavbar}
+							>
+								Contact Us
+							</NavLink>
+							<div className="navlink-image-container">
+								<AiFillCaretUp />
+							</div>
+						</li>
+					</ul>
```

Comment:
Suggestion: To reduce the return render repetition and keep it DRY:

```suggestion
					<ul>
						{navLinks.map((navLink) => {
							return (
								<li key={navLink.id}>
									<NavLink
										to={navLink.to}
										className="navigation-link"
										onClick={closeNavbar}
									>
										{navLink.text}
									</NavLink>
									<div className="navigation-link-icon-container">
										<AiFillCaretUp />
									</div>
								</li>
							);
						})}
					</ul>
```